Overview.src.html

```diff
@@ -1022,38 +1023,58 @@ <h3 id="request">Request</h3>
-    </ol>
+     <li><p><span data-anolis-spec=dom title=concept-event-fire>Fire an event</span> named
+     <code title=event-xhr-readystatechange>readystatechange</code>.
 
-    <p>To <span data-anolis-spec=fetch>process response body</span> for
-    <var title>response</var>, run these subsubsteps:
+     <li><p>If <span title=concept-XMLHttpRequest-state>state</span> is not
+     <i>headers received</i>, return.
 
-    <ol>
-     <li><p>If not roughly 50ms have passed since these subsubsteps were last invoked,
-     terminate these subsubsteps.
+     <li><p>If <var>response</var>'s
+     <span data-anolis-spec=fetch title=concept-response-body>body</span> is null,
+     run <span>handle response end-of-file</span> and return.
 
-     <li><p><span>Handle errors</span> for <var title>response</var>.
+     <li>
+      <p>Let <var>reader</var> be the result of
+      <span data-anolis-spec=fetch title=concept-get-reader>getting a reader</span> from
+      <var>response</var>'s <span data-anolis-spec=fetch title=concept-response-body>body</span>'s
+      <span data-anolis-spec=fetch title=concept-body-stream>stream</span>.
 
-     <li><p>If <span>response</span> is a
-     <span data-anolis-spec=fetch title=concept-network-error>network error</span>,
-     return.
+      <p><span class="note no-backref">This operation will not throw an exception.</span>
 
-     <li><p>If <span title=concept-XMLHttpRequest-state>state</span> is
-     <i title>headers received</i>, set
-     <span title=concept-XMLHttpRequest-state>state</span> to <i title>loading</i> and
-     <span data-anolis-spec=dom title=concept-event-fire>fire an event</span> named <code title=event-xhr-readystatechange>readystatechange</code>.
-
-     <li><p><span title=concept-event-fire-progress>Fire a progress event</span> named <code title="event-xhr-progress">progress</code> with <var title>response</var>'s
-     <span data-anolis-spec=fetch title=concept-response-body>body</span>'s
-     <span data-anolis-spec=fetch title=concept-body-transmitted>transmitted</span> and
-     <var title>response</var>'s
-     <span data-anolis-spec=fetch title=concept-response-body>body</span>'s
-     <span data-anolis-spec=fetch title=concept-body-length>length</span>.
-    </ol>
+     <li>
+      <p>Let <var>read</var> be the result of calling
+      <code data-anolis-spec=streams>ReadFromReadableStreamReader</code>(<var>reader</var>).
+
+      <p>When <var>read</var> is fulfilled with an object whose <code>done</code> property is false
+      and whose <code>value</code> property is a <code>Uint8Array</code> object, run these
+      subsubsubsteps and then run the above subsubstep again:
+
+      <ol>
+       <li><p>Append the <code>value</code> property to <span>received bytes</span>.
+
+       <li><p>If not roughly 50ms have passed since these subsubsubsteps were last invoked,
+       terminate these subsubsubsteps.
```

Comment:
"then terminate" (I guess you get the idea, if you could apply this consistently to the statements you changed that'd be great.)